app/views/shared/_footer.html.haml

```diff
@@ -143,8 +143,11 @@
           %div
             = t :footer_legal_data_text
             = t :footer_legal_data_call
-            %a{'cookies-policy-modal'=> true}
+            %a{'cookies-policy-modal'=> true, 'cookies-banner'=> @show_cookies_banner}
               = t :footer_legal_data_cookies_policy
+        %p.text-small
+          %a{ng: { controller:'CookiesBannerCtrl', click: 'deleteCookiesConsent()' }}
+            = 'delete consent cookie (for manual testing only)'
```

Comment:
This doesn't look like something that should be in production code. Did you forget to remove it?

Reply:
(I am pretty sure I have replied to this comment already somewhere else, sorry if it's duplicated)
I left it on purpose so that this can be manually tested. Otherwise I dont know how testers will be able to accept cookies more than once to validate everything is fine. Do you know how we can do that? Thanks!

Reply:
I didn't forget about this. I just thought we would need this for manual testing. So that manual testers could delete the consent_cookie. BUT I just realised it's extremelly easy to delete a cookie in chromes dev tools on the "Application" tab. I have deleted this code now.

Reply:
I also find it extremely useful to open a private window, test something and then close it again. All cookies of the private window are gone after closing it. You can even do that on mobile.